apps/a1/Dockerfile

```
@@ -0,0 +1,33 @@
+# Pre-requisite: uv sync --all-packages
+
+# Build
+# We use the same image as the runtime base image so that Python
+# version and location matches. Might save on data transfer costs.
+FROM us-west1-docker.pkg.dev/serverless-runtimes/google-22/runtimes/python313 AS builder
+
+# Sorta hacky solution to being able to write to /workspace
+USER root
```

Hadolint (2.12.0)

[warning] Last USER should not be root

(DL3002)